portal/views/patients.py

```diff
@@ -95,8 +95,25 @@ def patients_root():
                      UserIntervention.intervention_id.in_(ui_list)))
         patients = patients.union(ui_patients)
 
+    #get assessment status only if it is needed as specified by config
+    patient_list_addl_fields = current_app.config.get('PATIENT_LIST_ADDL_FIELDS')
+    if 'status' in patient_list_addl_fields:
```

Comment:
no need for temp variable 'patient_list_addl_fiels' - do in one line:
`if 'status' in current_app.config.get('PATIENT_LIST_ADDL_FIELDS'):`